samples/chair.js

```diff
@@ -0,0 +1,33 @@
+AFRAME.registerComponent("chair", {
```

Comment:
Is chair still an appropriate name for this?

Reply:
not at all lol. It was the name we used in all the other samples so far so i didnt change it. I will change now